snippets/resolver/named.conf

```
@@ -1,28 +1,48 @@
 options {
-  directory "/var/cache/bind";
-  listen-on { any; };
-  listen-on-v6 { any; };
-  pid-file "/run/named/named.pid";
+directory "/var/cache/bind";
+allow-recursion {
+192.168.0.0/24;
+127.0.0.1/32;
 };
-
-zone "2tm1-admin-sys2.ephec.austreelis.net" IN {
-  type master;
-  file "/etc/bind/2tm1-admin-sys2.ephec.austreelis.net.zone";
+// Configure the IPs to listen on here.
+listen-on port 53 {any;};
+listen-on-v6 { none; };
```

Fix indentation. Also, I'm pretty sure we don't want to opt *out* of IPv6 (even if we don't use it for now). Enabling IPv6 should probably be done on the network/routing level, and we should let `named` use it *if available* in all cases.

```suggestion
  listen-on-v6 { any; };
```